bin/ch_misc.c

```diff
@@ -471,12 +487,22 @@ void msgv(enum log_level level, const char *file, int line, int errno_,
    if (fmt == NULL)
       fmt = "please report this bug";
 
-   vfprintf(stderr, fmt, ap);
-   if (errno_)
-      fprintf(stderr, ": %s (%s:%d %d)\n",
-              strerror(errno_), file, line, errno_);
-   else
-      fprintf(stderr, " (%s:%d)\n", file, line);
+   char *ap_msg = "";
```

Comment:
I don't like to declare variables in the middle of a scope because they're valid for the entire scope, i.e., also valid in the code above before you get to the declaration.